<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Visa_Form_Screen</class>
 <widget class="QDialog" name="Visa_Form_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Visa Application Form</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgb(70, 153, 123);
    font-size: 20px;
    font-weight: 600;
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Roboto&quot;, &quot;Helvetica Neue&quot;, Arial, sans-serif;
    qproperty-alignment: AlignHCenter;
}
</string>
   </property>
   <property name="text">
    <string>Visa Application Form</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Full Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;
}

/* Hover effect */
QLineEdit:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QLineEdit:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Father Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;
}

/* Hover effect */
QLineEdit:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QLineEdit:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;
}

/* Hover effect */
QLineEdit:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QLineEdit:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Phone Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>143</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;

}

/* Hover effect */
QTextEdit:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QTextEdit:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;
}

/* Hover effect */
QLineEdit:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QLineEdit:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>CNIC No:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;
    

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0 ;
    border-radius: 6px;
	
}

/* Hover effect */
QComboBox:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QComboBox:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Nationality:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Valid CNIC:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;
    minimum_height: 20;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;

    /* Padding (horizontal only to keep default height) */
    padding: 0px 10px;
}

/* Hover effect */
QComboBox:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QComboBox:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Passport No:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Valid Passport:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;
}

/* Hover effect */
QLineEdit:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QLineEdit:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;
    minimum_height: 20;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;

    /* Padding (horizontal only to keep default height) */
    padding: 0px 10px;
}

/* Hover effect */
QComboBox:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QComboBox:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Destination:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;
    

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0 ;
    border-radius: 6px;
	
}

/* Hover effect */
QComboBox:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QComboBox:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="suffix">
    <string> Day(s)</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>365</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Criminal Record:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;
    minimum_height: 20;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;

    /* Padding (horizontal only to keep default height) */
    padding: 0px 10px;
}

/* Hover effect */
QComboBox:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QComboBox:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Any Prior Overstay:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;
    minimum_height: 20;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;

    /* Padding (horizontal only to keep default height) */
    padding: 0px 10px;
}

/* Hover effect */
QComboBox:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QComboBox:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;
    minimum_height: 20;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;

    /* Padding (horizontal only to keep default height) */
    padding: 0px 10px;
}

/* Hover effect */
QComboBox:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QComboBox:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;
    minimum_height: 20;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;

    /* Padding (horizontal only to keep default height) */
    padding: 0px 10px;
}

/* Hover effect */
QComboBox:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QComboBox:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Any Deportation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>500</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Previous Visa Denial:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Tax Payer:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;
    minimum_height: 20;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;

    /* Padding (horizontal only to keep default height) */
    padding: 0px 10px;
}

/* Hover effect */
QComboBox:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QComboBox:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_10">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;
    minimum_height: 20;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;

    /* Padding (horizontal only to keep default height) */
    padding: 0px 10px;
}

/* Hover effect */
QComboBox:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QComboBox:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>540</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Ties to Home Country:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Monthly Income:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>580</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Estimated Travel Cost:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>580</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="maximum">
    <double>100000000000000000000.000000000000000</double>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>580</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000000000000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>620</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6fdc8c,
        stop:1 #2fa866
    );
    color: white;
    border: none;
    font-size: 15px;
    font-weight: 600;
}

QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #7fe09a,
        stop:1 #34b673
    );
}

QPushButton:pressed {
    background: #2fa866;
}
</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 12px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Stay Duration:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
